Added 9 faculty CS profs from THWS. All relevant links provided.

i-yam opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

i-yam committed:
Update country-info.csv

Adding THWS,europe,de 

here is the information on the right to grand PhD
https://www.thws.de/service/news-presse/pressemeldungen/thema/thws-erhaelt-promotionsrecht/

i-yam committed:
Update csrankings-f.csv

Adding Frank Deinzer and Frank-Michael Schleif from THWS

i-yam committed:
Update csrankings-i.csv

Adding Ivan P. Yamshchikov, THWS

i-yam committed:
Update csrankings-m.csv

Adding Magda Gregorova, THWS

i-yam committed:
Update csrankings-p.csv

adding Pascal Meissner,THWS

i-yam committed:
Update csrankings-a.csv

Added Andreas Lehrmann and Anne Hess from THWS

i-yam committed:
Update csrankings-c.csv

Added Christian Bachmeir from THWS

i-yam committed:
Update csrankings-d.csv

Added Dominik Seuß from THWS

i-yam committed:
Update csrankings-d.csv

Added Dorit Borrmann from THWS

i-yam committed:
Update csrankings-k.csv

added Karsten Huffstadt from THWS

i-yam committed:
Update csrankings-m.csv

Added Martin Storath from THWS

i-yam committed:
Update csrankings-v.csv

added Volker Willert from THWS

i-yam committed:
Update csrankings-t.csv

added Tobias Kaupp from THWS

i-yam committed:
Merge branch 'gh-pages' into gh-pages

i-yam committed:
Update csrankings-v.csv

updated a link for Volker Willert (previous quit with error 255

i-yam committed:
Update csrankings-t.csv

updated a link for tobias kaupp

i-yam reviewed (commented):
had issues with long faculty webpages, tried url shortener instead

i-yam committed:
Update csrankings-d.csv

changed spelling for domnik seuss

i-yam committed:
Update csrankings-d.csv

provided a short URL for faculty page for Dorit Borrmann

i-yam reviewed (commented):
updated spelling for dominik seuss and shortened the usr for dorit borrmann

i-yam committed:
Update csrankings-d.csv

added alias for dominik seuss

i-yam reviewed (commented):
added alias for domninik seuss

i-yam committed:
Update csrankings-d.csv

deleted Dominik Seuss due to some internal problem in autocheck

i-yam reviewed (commented):
deleted dominik Seuss dues to alias conflicts

i-yam committed:
Merge branch 'gh-pages' into gh-pages

emeryberger commented:
Does not appear to be a PhD-granting institution.
https://fiw.thws.de/en/our-offers/

i-yam commented:
No, we had discussed it already (https://github.com/emeryberger/CSrankings/pull/7354#event-14689419911). The right to grant PhDs was given in autumn 2023, so it's new.

Once again, here is the proof that we give PhDs:
https://www.thws.de/service/news-presse/pressemeldungen/thema/thws-erhaelt-promotionsrecht/

THWS is already in the rankings, this request is just to add more colleagues who publish in CS.

i-yam commented:
Please, reopen

emeryberger commented on csrankings-v.csv:
```diff
@@ -496,6 +496,7 @@ Volker Sorge,University of Birmingham,https://www.birmingham.ac.uk/staff/profile
 Volker Steinhage,University of Bonn,https://net.cs.uni-bonn.de/wg/intelligent-vision-systems/staff/head-of-work-group/volker-steinhage,2zyaIpEAAAAJ
 Volker Tresp,LMU Munich,http://www.dbs.ifi.lmu.de/~tresp,xIJHTUwAAAAJ
 Volker Turau,Hamburg University of Technology,https://www.ti5.tuhh.de/staff/turau,YuVtpLsAAAAJ
+Volker Willert,THWS,https://shorturl.at/PKGAq,e4LRaz0AAAAJ
```
dead link

emeryberger commented on csrankings-t.csv:
```diff
@@ -740,6 +740,7 @@ Tobias Glasmachers,Ruhr-University Bochum,https://www.ini.rub.de/the_institute/p
 Tobias Hossfeld,University of Würzburg,http://www.comnet.informatik.uni-wuerzburg.de/team/mitarbeiter/hossfeld,AdGp9e0AAAAJ
 Tobias Hoßfeld,University of Würzburg,http://www.comnet.informatik.uni-wuerzburg.de/team/mitarbeiter/hossfeld,AdGp9e0AAAAJ
 Tobias Höllerer,Univ. of California - Santa Barbara,http://www.cs.ucsb.edu/~holl,008lr2cAAAAJ
+Tobias Kaupp,THWS,https://shorturl.at/gSnPU,auMQ1JQAAAAJ
```
Dead link. Please do not use link shorteners.

emeryberger commented on csrankings-d.csv:
```diff
@@ -1489,6 +1489,7 @@ Dorian C. Arnold,Emory University,https://cs.emory.edu/~darnold,2jEFz5EAAAAJ
 Dorien Herremans,SUTD,http://dorienherremans.com,Hp5W5f0AAAAJ
 Doris L. Carver,Louisiana State University,https://www.lsu.edu/eng/cse/people/faculty/carver.php,NOSCHOLARPAGE
 Dorit Aharonov,Hebrew University of Jerusalem,http://www.cs.huji.ac.il/~doria,hTBcSYQAAAAJ
+Dorit Borrmann,THWS,https://shorturl.at/1MjRG,R07G1FQAAAAJ
```
Dead link; do not use link shorteners.

i-yam committed:
Update csrankings-d.csv

i-yam committed:
Update csrankings-t.csv

added the link to the page with all robotics professors. The individual links somehow do not pass the checks

i-yam committed:
Update csrankings-v.csv

i-yam committed:
Update csrankings-d.csv

added the link to all robotics profs (the direct ones somehow do not pass automated checks)

i-yam reviewed (commented):
replaced the shortened links with the link to the robotics institute people page (somehow the links to the individual pages do not pass the automated checks, I guess they are too long)

i-yam committed:
Merge branch 'emeryberger:gh-pages' into gh-pages

emeryberger committed:
Merge branch 'gh-pages' into gh-pages
